packages/icon/README.md

```diff
@@ -1,17 +1,16 @@
-This is currently not working as automatically as we'd like.  If there is another large quantity of new icons or old icon adjustments, this will probably want to be improved before then.
+This is currently not working as automatically as we'd like. If there is another large quantity of new icons or old icon adjustments, this will probably want to be improved before then.
 
 ## To Produce New Svgs
 
 1. Draw SVGs in Sketch in the same file as the other icons (see Dmitry for file)
-1. Export with the [svgo](https://github.com/BohemianCoding/svgo-compressor) plugin.  It doesn't seem to make a ton of difference (xlink is used either way, and that's unfortunate)
+1. Export with the [svgo](https://github.com/BohemianCoding/svgo-compressor) plugin. It doesn't seem to make a ton of difference (xlink is used either way, and that's unfortunate)
 
 ## To Add New Icon Svgs
 
-1. Name icons *.icon.svg
+1. Name icons \*.icon.svg
 1. Put in the `src/svg` directory
 1. Add `role="img" and aria-label="<iconId here> icon"` to the root `svg` element of each icon
 1. Manually refactor the svg to not use xlink/defs.
-1. Run `npm run build` in the `packages/icon` directory (new `*.svg.dist.js` files will generate new icon ids automatically)
+1. Run `npm run build` in the `packages/icon` directory (new `*.svg.js` files will generate new icon ids automatically)
```

Comment:
Not a huge deal, so please don't revert, but I liked the `dist` inclusion in the file name so that it was clear it was from the build and not src.  What was your thinking on the name change?

Reply:
i actually like it better that way now that i think of it. i changed it back